commands/serve/web/components/Cell.jsx

```diff
@@ -152,13 +165,7 @@ export default function ({ id, expandable, minHeight }) {
             title="Open in single render view"
             className={classes.secondaryIcon}
             disabled={!model}
-            href={
-              model
-                ? `${document.location.href.replace(/\/dev\//, '/render/')}${
-                    window.location.search ? '&' : '?'
-                  }object=${model.id}&theme=${currentThemeName}&language=${language}`
-                : ''
-            }
+            href={singleRenderUrl()}
```

Comment:
The refactor is really good. But it is not relevant to this PR. Would you mind doing it in another PR?

Reply:
I would argue that it is relevant to this PR. The alternative to not doing this refactoring would be to add duplicated code.

Reply:
ahh, I see. singleRenderUrl is used in openInSingleRender.